go/kbfs/libkbfs/kbfs_ops_test.go

```diff
@@ -258,23 +257,23 @@ func kbfsOpsInitNoMocks(t *testing.T, users ...kbname.NormalizedUsername) (
 	return config, session.UID, ctx, cancel
 }
 
-func kbfsTestShutdownNoMocks(t *testing.T, config *ConfigLocal,
-	ctx context.Context, cancel context.CancelFunc) {
+func kbfsTestShutdownNoMocks(ctx context.Context, t *testing.T,
```

Comment:
You didn't change this but would be nice to move these params down to the next line too, while you're here.